src/main/resources/db/changelog/alert/5.1.0/changelog-configuration-stored-procedures.xml

```diff
@@ -0,0 +1,19 @@
+<databaseChangeLog xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.3.xsd">
+    <changeSet author="psantos" id="1565285011567-1">
+        <createProcedure
+                dbms="h2"
+                relativeToChangelogFile="true"
+                schemaName="ALERT">
+            CREATE ALIAS REGISTER_DESCRIPTOR AS $$
+                java.lang.Void registerDescriptor(final java.sql.Connection connection, final java.lang.String descriptorName, java.lang.String type) throws java.sql.SQLException {
```

Comment:
This is minor, but some of the method parameters are listed as `final` and some are not.